cpu/esp32/Makefile.dep

```diff
@@ -1,5 +1,8 @@
 # additional modules dependencies
 
+# thin archives trigger a reboot loop - see #12258, #12035
```

Comment:
Since there is no module dependency, it should be set in `Makefile.include`, e.g., below the `CFLAGS` settings.

Reply:
Moved it.